sdks/java/io/hdfs/pom.xml

```diff
@@ -64,6 +64,20 @@
       <scope>provided</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.avro</groupId>
+      <artifactId>avro-mapred</artifactId>
+      <version>1.7.7</version>
```

Comment:
+1 (or no version at all as it can come from the dependencyManagement of the parent pom).